Dockerfile

```diff
@@ -74,6 +69,10 @@ RUN chmod 0755 bin/*
 COPY --chown=ruby:ruby --from=build /usr/local/bundle /usr/local/bundle
 COPY --chown=ruby:ruby --from=build /app /app
 
+RUN mkdir -p "/app/tmp/" && chown ruby:ruby "/app/tmp/"
```

Comment:
If it's easier we could instead have a `/tmp/.keep` file in the repo (so we don't have to create the folder in the Dockerfile)

Reply:
I think creating the directory here is the right place for it. It existing is an infrastructure concern, and having an all-but-empty directory in the source repo leaves us in a position where somebody can delete the directory with no knowledge of why it's there and be left with a mysterious failure to debug.